webapp/store/views.py

```diff
@@ -231,6 +234,33 @@ def brand_search_snap():
             status_code,
         )
 
+    def get_file(file):
+
+        try:
+            import os
+
+            with open(
+                os.path.join(flask.current_app.root_path, file), "r"
+            ) as stream:
+                data = yaml.load(stream)
+        except Exception:
+            data = None
+
+        return data
+
+    @store.route("/publisher/jetbrains")
+    def publisher_details_jetbrains():
```

Comment:
@tbille @jpmartinspt We would need some backend look on this. It was quickly glued together with bits from other parts of the app (we read YAML for first snap flow), but I'm not sure if that is good to go as it is.

Also this endpoint needs a test not to decrease coverage.